cmd/tuf/import_key.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/flynn/go-docopt"
+	"github.com/theupdateframework/go-tuf"
+	"github.com/theupdateframework/go-tuf/data"
+	"github.com/theupdateframework/go-tuf/pkg/keys"
+)
+
+func init() {
+	register("import-key", cmdImportKey, `
+usage: tuf import-key [--expires=<days>] <role> <file>
+
+Import an existing signing key for the given role.
+
+The key will be imported from the file and added to the "keys" directory with
+filename pattern "ROLE-KEYID.json". The root metadata file will also be staged
+with the addition of the key's ID to the role's list of key IDs.
+
+Options:
+  --expires=<days>   Set the root metadata file to expire <days> days from now.
+`)
+}
+
+func cmdImportKey(args *docopt.Args, repo *tuf.Repo) error {
```

Comment:
Would prefer to see a `repo.ImportKey` that took a `PublicKey` for the key and `role` for the role so then we could test it a little better (long-run I'd like to have actual CLI tests but short-run we do what we can :) ).